migrations/sqls/20231102122939-delete-broken-licence-up.sql

```diff
@@ -0,0 +1,112 @@
+/* This query is deleting a bad licence data that has a carriage return at the end of it */
+DO $$
+    BEGIN
+        IF EXISTS
+            ( SELECT 1
+              FROM   information_schema.tables
+              WHERE  table_schema = 'crm_v2'
+              AND    table_name = 'document_roles'
+            )
+        THEN
+          DELETE FROM "crm_v2"."document_roles" WHERE document_id IN (
+            SELECT document_id FROM "crm_v2"."documents" WHERE document_ref LIKE 'NW/072/0417/002/R01' || CHR(13)
```

Comment:
This is my bad as I was on the call with you when you worked on this but I think where you have used `WHERE licence_ref LIKE 'NW/072/0417/002/R01' || CHR(13)` you can use an `=` instead of `LIKE` so it's `WHERE licence_ref = 'NW/072/0417/002/R01' || CHR(13)`. That should work for all of them

Reply:
All changed :)

Reply:
All changed :)